src/core/types/signatures/ed25519_signature.rs

```diff
@@ -57,6 +57,15 @@ impl Ed25519Signature {
     pub fn to_hex(&self) -> String {
         hex_encode(self.to_bytes())
     }
+
+    pub fn from_hex(hex: String) -> Result<Self> {
+        // We want to ALWAYS go via `try_from(slice: &[u8])` since validates the EC point (`ScryptoEd25519PublicKey` doesn't!)
```

Comment:
copy paste error comment? this relates to signatures, not public keys

Reply:
But this code is in Ed25519_SIGNATURE, or you mean something else?